examples/nwc/wallet-service/get-info.ts

```
@@ -43,6 +43,7 @@ const keypair = new NWCWalletServiceKeyPair(
 
 const unsub = await walletService.subscribe(keypair, {
   getInfo: () => {
+    console.info("get_info requested");
```

ESLint

[error] Unexpected console statement.

(no-console)